core/metacling/src/TCling.cxx

```diff
@@ -1418,6 +1418,7 @@ TCling::TCling(const char *name, const char *title, const char* const argv[])
       // discover modules without having to should thisroot.sh and should fix
       // gnuinstall.
       Paths.push_back(TROOT::GetLibDir().Data());
+      Paths.push_back(TROOT::GetBinDir().Data());
```

Comment:
On windows the dlls are in the bin directory and on unix in the lib directory. We should add those depending on `R__WIN32` macro.

Reply:
I updated the commit